host/libhackrf/src/hackrf.h

```diff
@@ -136,305 +778,939 @@ enum operacake_ports {
+ * 
+ * @param device device to configure
+ * @param value number of samples to wait before auto-stopping
+ * @return @ref HACKRF_SUCCESS on success or @ref hackrf_error variant
+ * @ingroup streaming
+ */
 extern ADDAPI int ADDCALL hackrf_set_rx_overrun_limit(
 	hackrf_device* device,
 	uint32_t value);
 
-/* return HACKRF_TRUE if success */
+/**
+ * Query device streaming status
+ * 
+ * @param device device to query
+ * @return @ref HACKRF_TRUE if the device is streaming, else one of @ref HACKRF_ERROR_STREAMING_THREAD_ERR, @ref HACKRF_ERROR_STREAMING_STOPPED or @ref HACKRF_ERROR_STREAMING_EXIT_CALLED
+ * @ingroup streaming
+ */
 extern ADDAPI int ADDCALL hackrf_is_streaming(hackrf_device* device);
 
+/**
+ * Directly read the registers of the MAX2837 transceiver IC
+ * 
+ * Intended for debugging purposes only!
+ * 
+ * @param[in] device device to query
+ * @param[in] register_number register number to read
+ * @param[out] value value of the specified register
+ * @return @ref HACKRF_SUCCESS on success or @ref hackrf_error variant
+ * @ingroup debug
+ */
 extern ADDAPI int ADDCALL hackrf_max2837_read(
 	hackrf_device* device,
 	uint8_t register_number,
 	uint16_t* value);
 
+/**
+ * Directly write the registers of the MAX2837 transceiver IC
+ * 
+ * Intended for debugging purposes only!
+ * 
+ * @param device device to write
+ * @param register_number register number to write
+ * @param value value to write in the specified register
+ * @return @ref HACKRF_SUCCESS on success or @ref hackrf_error variant
+ * @ingroup debug
+ */
 extern ADDAPI int ADDCALL hackrf_max2837_write(
 	hackrf_device* device,
 	uint8_t register_number,
 	uint16_t value);
 
+/**
+ * Directly read the registers of the Si5351C clock generator IC
+ * 
+ * Intended for debugging purposes only!
+ * 
+ * @param[in] device device to query
+ * @param[in] register_number register number to read
+ * @param[out] value value of the specified register
+ * @return @ref HACKRF_SUCCESS on success or @ref hackrf_error variant
+ * @ingroup debug
+ */
 extern ADDAPI int ADDCALL hackrf_si5351c_read(
 	hackrf_device* device,
 	uint16_t register_number,
 	uint16_t* value);
 
+/**
+ * Directly write the registers of the Si5351 clock generator IC
+ * 
+ * Intended for debugging purposes only!
+ * 
+ * @param[in] device device to write
+ * @param[in] register_number register number to write
+ * @param[out] value value to write in the specified register
+ * @return @ref HACKRF_SUCCESS on success or @ref hackrf_error variant
+ * @ingroup debug
+ */
 extern ADDAPI int ADDCALL hackrf_si5351c_write(
 	hackrf_device* device,
 	uint16_t register_number,
 	uint16_t value);
 
+/**
+ * Set baseband filter bandwidth
+ * 
+ * Possible values: 1.75/2.5/3.5/5/5.5/6/7/8/9/10/12/14/15/20/24/28MHz, default <= 0.75 * sample_rate_hz
```

Comment:
This list of values is hard to read in this format. 
```suggestion
 * Possible values: 1.75, 2.5, 3.5, 5, 5.5, 6, 7, 8, 9, 10, 12, 14, 15, 20, 24, or 28 MHz.
```
The default could be better rendered as `\f$ \le 0.75 \cdot F_s\f$`, both here and elsewhere.